nservicebus/sqlserver/index.md

```diff
@@ -17,7 +17,7 @@ The SQL Server transport implements a message queueing mechanism on top of Micro
 
 ## How it works
 
-The SQL Server transport is a hybrid queueing system which is neither store-and-forward (like MSMQ) nor a broker (like RabbitMQ). It treats the SQL Server as a storage infrastructure for the queues. The queueing logic itself is implemented and executed as part of the transport code running in an NServiceBus endpoint.
+SQL Server provides a central place to store queues and messages but the message queuing implementation resides entirely within the endpoint process. The SQL Server transport is best thought of as a brokered transport like RabbitMQ rather than [store-and-forward](/nservicebus/architecture/principles.md#drilling-down-into-details-store-and-forward-messaging) transport such as MSMQ.
```

Comment:
Wouldn't it be simpler to say that in SQL Server transport we use SQL Server only to store queues and messages? If we really want to be explicit then we can say that we don't use queueing services provided by it or that we implement queueing logic ourselves.